(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 100k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 1 ',)
head -c 100k /dev/urandom > /tmp/tmp9l129dip
Data Size: 100k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmp9l129dip  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 30 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.330737	0.000000	0.330843		0.957280	0		200		102400		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 200k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 1 ',)
head -c 200k /dev/urandom > /tmp/tmp99o727n1
Data Size: 200k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmp99o727n1  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 30 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319417	0.000000	0.319521		1.113936	0		200		204800		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 300k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 1 ',)
head -c 300k /dev/urandom > /tmp/tmp7nqbhpv0
Data Size: 300k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmp7nqbhpv0  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 30 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319419	0.000000	0.319528		1.265277	0		200		307200		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 400k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 1 ',)
head -c 400k /dev/urandom > /tmp/tmpzizfztq9
Data Size: 400k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpzizfztq9  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 30 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319241	0.000000	0.319350		1.538984	0		200		409600		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 500k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 1 ',)
head -c 500k /dev/urandom > /tmp/tmpvalkdg_5
Data Size: 500k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpvalkdg_5  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 30 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.330600	0.000000	0.330727		1.645507	0		200		512000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 600k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 1 ',)
head -c 600k /dev/urandom > /tmp/tmpzwy190ak
Data Size: 600k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpzwy190ak  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 30 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.330748	0.000000	0.330853		1.856926	0		200		614400		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 700k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 1 ',)
head -c 700k /dev/urandom > /tmp/tmpkejno0c6
Data Size: 700k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpkejno0c6  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 30 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319549	0.000000	0.319652		1.998385	0		200		716800		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 800k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 1 ',)
head -c 800k /dev/urandom > /tmp/tmpw9tm2vx2
Data Size: 800k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpw9tm2vx2  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 30 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.324619	0.000000	0.324725		2.065832	0		200		819200		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 900k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 1 ',)
head -c 900k /dev/urandom > /tmp/tmp3k709sky
Data Size: 900k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmp3k709sky  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 30 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319249	0.000000	0.319354		2.187800	0		200		921600		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 1000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 1 ',)
head -c 1000k /dev/urandom > /tmp/tmp6hxalo66
Data Size: 1000k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmp6hxalo66  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 30 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319295	0.000000	0.319421		2.408137	0		200		1024000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 2000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 1 ',)
head -c 2000k /dev/urandom > /tmp/tmp33n2q1wb
Data Size: 2000k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmp33n2q1wb  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 60 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319687	0.000000	0.319796		4.261400	0		200		2048000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 3000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 1 ',)
head -c 3000k /dev/urandom > /tmp/tmpwesn_sbt
Data Size: 3000k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpwesn_sbt  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 90 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.325485	0.000000	0.325597		5.675933	0		200		3072000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 4000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 1 ',)
head -c 4000k /dev/urandom > /tmp/tmpfoqrd4g4
Data Size: 4000k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpfoqrd4g4  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 120 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.331117	0.000000	0.331227		7.095736	0		200		4096000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 5000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 1 ',)
head -c 5000k /dev/urandom > /tmp/tmp90dd01r1
Data Size: 5000k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmp90dd01r1  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 150 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319324	0.000000	0.319436		6.972408	0		200		5120000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 6000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 1 ',)
head -c 6000k /dev/urandom > /tmp/tmpos54vm4b
Data Size: 6000k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpos54vm4b  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 180 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319353	0.000000	0.319459		9.548165	0		200		6144000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 7000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 1 ',)
head -c 7000k /dev/urandom > /tmp/tmp7hbg7aik
Data Size: 7000k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmp7hbg7aik  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 210 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319485	0.000000	0.319595		8.999234	0		200		7168000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 8000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 1 ',)
head -c 8000k /dev/urandom > /tmp/tmpws4fgtco
Data Size: 8000k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpws4fgtco  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 240 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319422	0.000000	0.319536		11.451747	0		200		8192000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 9000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 1 ',)
head -c 9000k /dev/urandom > /tmp/tmp61delpwz
Data Size: 9000k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmp61delpwz  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 270 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.330051	0.000000	0.330162		12.396917	0		200		9216000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 10000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 1 ',)
head -c 10000k /dev/urandom > /tmp/tmpegwyl7c4
Data Size: 10000k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpegwyl7c4  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 300 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 10000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 10 ',)
head -c 10000k /dev/urandom > /tmp/tmpfkj1r85f
Data Size: 10000k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpfkj1r85f  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 300 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.330575	0.000000	0.330684		15.077296	0		200		10240000		18	
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 11000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 10 ',)
head -c 11000k /dev/urandom > /tmp/tmp110yhs3i
Data Size: 11000k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmp110yhs3i  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 330 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.330793	0.000000	0.330899		16.616044	0		200		11264000		18	
0.158691	0.000000	0.158794		14.640068	0		200		11264000		18	
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 12000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 1 ',)
head -c 12000k /dev/urandom > /tmp/tmp_6932ob1
Data Size: 12000k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmp_6932ob1  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 360 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319398	0.000000	0.319506		16.609568	0		200		12288000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 13000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 1 ',)
head -c 13000k /dev/urandom > /tmp/tmpww0d74r7
Data Size: 13000k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpww0d74r7  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 390 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.324611	0.000000	0.324720		17.934556	0		200		13312000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 14000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 1 ',)
head -c 14000k /dev/urandom > /tmp/tmpmzsiypkt
Data Size: 14000k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpmzsiypkt  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 420 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319340	0.000000	0.319449		19.864809	0		200		14336000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 15000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 1 ',)
head -c 15000k /dev/urandom > /tmp/tmpuiuzaf5y
Data Size: 15000k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpuiuzaf5y  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 450 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319482	0.000000	0.319604		21.184353	0		200		15360000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 16000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 1 ',)
head -c 16000k /dev/urandom > /tmp/tmpegnrissz
Data Size: 16000k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpegnrissz  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 480 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.326229	0.000000	0.326337		22.237658	0		200		16384000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 17000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 1 ',)
head -c 17000k /dev/urandom > /tmp/tmp2fjmnuii
Data Size: 17000k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmp2fjmnuii  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 510 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319456	0.000000	0.319597		21.721079	0		200		17408000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 18000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 1 ',)
head -c 18000k /dev/urandom > /tmp/tmpwuftisd0
Data Size: 18000k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpwuftisd0  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 540 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319200	0.000000	0.319307		24.582180	0		200		18432000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 19000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 1 ',)
head -c 19000k /dev/urandom > /tmp/tmpbtnjkkb0
Data Size: 19000k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpbtnjkkb0  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 570 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319404	0.000000	0.319514		27.249733	0		200		19456000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 20000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 1 ',)
head -c 20000k /dev/urandom > /tmp/tmp2z_bdj1u
Data Size: 20000k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmp2z_bdj1u  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 600 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319314	0.000000	0.319426		27.650318	0		200		20480000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 22000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 1 ',)
head -c 22000k /dev/urandom > /tmp/tmpwxaehma4
Data Size: 22000k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpwxaehma4  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 660 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319866	0.000000	0.319973		30.710726	0		200		22528000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 24000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 1 ',)
head -c 24000k /dev/urandom > /tmp/tmpednrnvsq
Data Size: 24000k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpednrnvsq  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 720 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319295	0.000000	0.319403		33.424201	0		200		24576000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 26000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 1 ',)
head -c 26000k /dev/urandom > /tmp/tmpdgc391as
Data Size: 26000k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpdgc391as  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 780 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319291	0.000000	0.319400		37.328284	0		200		26624000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 28000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 1 ',)
head -c 28000k /dev/urandom > /tmp/tmpx5c678g9
Data Size: 28000k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpx5c678g9  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 840 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319693	0.000000	0.319799		840.001237	28		000		28672000		0	Operation timed out after 840001 milliseconds with 0 bytes received
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 30000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 1 ',)
head -c 30000k /dev/urandom > /tmp/tmpcu0rizye
Data Size: 30000k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpcu0rizye  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 900 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.325775	0.000000	0.325881		38.415671	0		200		30720000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 32000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 1 ',)
head -c 32000k /dev/urandom > /tmp/tmp6o4arko6
Data Size: 32000k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmp6o4arko6  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 960 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319454	0.000000	0.319564		44.621004	0		200		32768000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 34000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 1 ',)
head -c 34000k /dev/urandom > /tmp/tmpi7pk0qjy
Data Size: 34000k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpi7pk0qjy  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 1020 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319394	0.000000	0.319499		43.885551	0		200		34816000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 36000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 1 ',)
head -c 36000k /dev/urandom > /tmp/tmpuwticpcj
Data Size: 36000k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpuwticpcj  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 1080 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319346	0.000000	0.319454		47.711257	0		200		36864000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 38000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 1 ',)
head -c 38000k /dev/urandom > /tmp/tmpem6om385
Data Size: 38000k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpem6om385  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 1140 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319715	0.000000	0.319824		53.504870	0		200		38912000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 40000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 1 ',)
head -c 40000k /dev/urandom > /tmp/tmpvb21slyb
Data Size: 40000k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpvb21slyb  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 1200 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.324747	0.000000	0.324854		58.202956	0		200		40960000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 50000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 1 ',)
head -c 50000k /dev/urandom > /tmp/tmpmjzgh8z_
Data Size: 50000k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpmjzgh8z_  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 1500 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319448	0.000000	0.319556		70.001722	0		200		51200000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 28000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 1 ',)
head -c 28000k /dev/urandom > /tmp/tmpkq3r7nns
Data Size: 28000k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpkq3r7nns  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 840 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 28000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 1 ',)
head -c 28000k /dev/urandom > /tmp/tmph7uapfgu
Data Size: 28000k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmph7uapfgu  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 840 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.325695	0.000000	0.325805		39.636826	0		200		28672000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 100k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 9 ',)
head -c 100k /dev/urandom > /tmp/tmpv3go6sou
Data Size: 100k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpv3go6sou  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 30 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319643	0.000000	0.319753		0.945794	0		200		102400		18	
0.164575	0.000000	0.164676		0.790154	0		200		102400		18	
0.158653	0.000000	0.158775		0.784059	0		200		102400		18	
0.158623	0.000000	0.158729		0.783950	0		200		102400		18	
0.158605	0.000000	0.158710		0.783946	0		200		102400		18	
0.158921	0.000000	0.159028		0.784341	0		200		102400		18	
0.158612	0.000000	0.158712		0.784018	0		200		102400		18	
0.158595	0.000000	0.158698		0.783907	0		200		102400		18	
0.158699	0.000000	0.158805		0.784117	0		200		102400		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 200k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 9 ',)
head -c 200k /dev/urandom > /tmp/tmpbvyfyy1_
Data Size: 200k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpbvyfyy1_  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 30 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.326644	0.000000	0.326755		1.121347	0		200		204800		18	
0.158812	0.000000	0.158924		0.952568	0		200		204800		18	
0.158718	0.000000	0.158824		0.954434	0		200		204800		18	
0.164502	0.000000	0.164609		0.958620	0		200		204800		18	
0.164463	0.000000	0.164571		0.960517	0		200		204800		18	
0.164443	0.000000	0.164548		0.958406	0		200		204800		18	
0.164389	0.000000	0.164495		0.958270	0		200		204800		18	
0.164423	0.000000	0.164530		0.960330	0		200		204800		18	
0.158691	0.000000	0.158802		0.954275	0		200		204800		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 300k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 9 ',)
head -c 300k /dev/urandom > /tmp/tmpf9kd0wzy
Data Size: 300k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpf9kd0wzy  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 30 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319460	0.000000	0.319567		1.265573	0		200		307200		18	
0.158693	0.000000	0.158802		1.103647	0		200		307200		18	
0.158654	0.000000	0.158762		1.108960	0		200		307200		18	
0.158977	0.000000	0.159079		1.109110	0		200		307200		18	
0.164506	0.000000	0.164609		1.109660	0		200		307200		18	
0.158651	0.000000	0.158755		1.108776	0		200		307200		18	
0.158566	0.000000	0.158663		1.108641	0		200		307200		18	
0.164598	0.000000	0.164706		1.115199	0		200		307200		18	
0.164496	0.000000	0.164602		1.109714	0		200		307200		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 400k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 9 ',)
head -c 400k /dev/urandom > /tmp/tmpgr4gl_ie
Data Size: 400k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpgr4gl_ie  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 30 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.330860	0.000000	0.330963		1.550826	0		200		409600		18	
0.158727	0.000000	0.158839		1.378303	0		200		409600		18	
0.164469	0.000000	0.164592		1.383770	0		200		409600		18	
0.158723	0.000000	0.158833		1.378068	0		200		409600		18	
0.158816	0.000000	0.158921		1.377697	0		200		409600		18	
0.164508	0.000000	0.164618		1.383851	0		200		409600		18	
0.158697	0.000000	0.158801		1.378146	0		200		409600		18	
0.158711	0.000000	0.158823		1.378216	0		200		409600		18	
0.158662	0.000000	0.158769		1.377980	0		200		409600		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 500k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 9 ',)
head -c 500k /dev/urandom > /tmp/tmpzgox63y0
Data Size: 500k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpzgox63y0  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 30 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.325405	0.000000	0.325514		1.640681	0		200		512000		18	
0.158772	0.000000	0.158881		1.476877	0		200		512000		18	
0.164497	0.000000	0.164608		1.478268	0		200		512000		18	
0.158618	0.000000	0.158724		1.476635	0		200		512000		18	
0.164695	0.000000	0.164805		1.480724	0		200		512000		18	
0.164633	0.000000	0.164745		1.478557	0		200		512000		18	
0.158690	0.000000	0.158797		1.476944	0		200		512000		18	
0.164439	0.000000	0.164552		1.478488	0		200		512000		18	
0.164522	0.000000	0.164636		1.482929	0		200		512000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 600k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 9 ',)
head -c 600k /dev/urandom > /tmp/tmpsiia_y3b
Data Size: 600k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpsiia_y3b  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 30 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.324716	0.000000	0.324824		1.850514	0		200		614400		18	
0.164421	0.000000	0.164525		1.689648	0		200		614400		18	
0.158603	0.000000	0.158707		1.561341	0		200		614400		18	
0.164469	0.000000	0.164574		1.689512	0		200		614400		18	
0.164375	0.000000	0.164479		1.689486	0		200		614400		18	
0.158608	0.000000	0.158723		1.562762	0		200		614400		18	
0.158662	0.000000	0.158768		1.562720	0		200		614400		18	
0.158659	0.000000	0.158766		1.562599	0		200		614400		18	
0.164449	0.000000	0.164550		1.689509	0		200		614400		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 700k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 9 ',)
head -c 700k /dev/urandom > /tmp/tmpssqc_rq3
Data Size: 700k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpssqc_rq3  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 30 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319430	0.000000	0.319539		1.997918	0		200		716800		18	
0.158693	0.000000	0.158802		1.836232	0		200		716800		18	
0.164446	0.000000	0.164556		1.842596	0		200		716800		18	
0.158596	0.000000	0.158695		1.835729	0		200		716800		18	
0.158935	0.000000	0.159038		1.835840	0		200		716800		18	
0.164491	0.000000	0.164597		1.842762	0		200		716800		18	
0.158973	0.000000	0.159080		1.686524	0		200		716800		18	
0.164422	0.000000	0.164539		1.842424	0		200		716800		18	
0.158710	0.000000	0.158820		1.686279	0		200		716800		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 800k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 9 ',)
head -c 800k /dev/urandom > /tmp/tmpx6u50g6b
Data Size: 800k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpx6u50g6b  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 30 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.324393	0.000000	0.324501		2.108853	0		200		819200		18	
0.158855	0.000000	0.158958		1.861230	0		200		819200		18	
0.164456	0.000000	0.164564		1.867992	0		200		819200		18	
0.158883	0.000000	0.158990		1.915683	0		200		819200		18	
0.164355	0.000000	0.164457		1.921312	0		200		819200		18	
0.158720	0.000000	0.158844		1.861948	0		200		819200		18	
0.164454	0.000000	0.164560		1.868144	0		200		819200		18	
0.164556	0.000000	0.164662		1.900126	0		200		819200		18	
0.164397	0.000000	0.164504		1.867925	0		200		819200		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 900k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 9 ',)
head -c 900k /dev/urandom > /tmp/tmpqqbgu72i
Data Size: 900k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpqqbgu72i  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 30 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.325937	0.000000	0.326044		2.198261	0		200		921600		18	
0.158919	0.000000	0.159024		2.030648	0		200		921600		18	
0.158639	0.000000	0.158745		2.001988	0		200		921600		18	
0.158677	0.000000	0.158784		2.002100	0		200		921600		18	
0.164459	0.000000	0.164565		2.042476	0		200		921600		18	
0.158686	0.000000	0.158803		2.002201	0		200		921600		18	
0.164463	0.000000	0.164570		2.118045	0		200		921600		18	
0.158718	0.000000	0.158826		2.003816	0		200		921600		18	
0.164407	0.000000	0.164518		2.035982	0		200		921600		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 1000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 9 ',)
head -c 1000k /dev/urandom > /tmp/tmpg99wt8wd
Data Size: 1000k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpg99wt8wd  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 30 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319464	0.000000	0.319574		2.414408	0		200		1024000		18	
0.158774	0.000000	0.158886		2.229241	0		200		1024000		18	
0.158851	0.000000	0.158957		2.228980	0		200		1024000		18	
0.164389	0.000000	0.164495		2.257055	0		200		1024000		18	
0.158862	0.000000	0.158988		2.230262	0		200		1024000		18	
0.158741	0.000000	0.158845		2.229236	0		200		1024000		18	
0.158639	0.000000	0.158747		2.227909	0		200		1024000		18	
0.158707	0.000000	0.158814		2.228708	0		200		1024000		18	
0.158655	0.000000	0.158773		2.238731	0		200		1024000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 2000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 9 ',)
head -c 2000k /dev/urandom > /tmp/tmp_9qc5ebq
Data Size: 2000k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmp_9qc5ebq  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 60 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319334	0.000000	0.319449		4.292122	0		200		2048000		18	
0.164498	0.000000	0.164607		4.102204	0		200		2048000		18	
0.158660	0.000000	0.158770		3.585006	0		200		2048000		18	
0.158796	0.000000	0.158910		3.751353	0		200		2048000		18	
0.158684	0.000000	0.158792		3.752247	0		200		2048000		18	
0.158698	0.000000	0.158805		3.583639	0		200		2048000		18	
0.158817	0.000000	0.158929		3.584676	0		200		2048000		18	
0.158611	0.000000	0.158719		3.752186	0		200		2048000		18	
0.164533	0.000000	0.164644		4.105147	0		200		2048000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 3000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 9 ',)
head -c 3000k /dev/urandom > /tmp/tmp2of6aiob
Data Size: 3000k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmp2of6aiob  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 90 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.326651	0.000000	0.326759		5.675052	0		200		3072000		18	
0.158632	0.000000	0.158736		5.509143	0		200		3072000		18	
0.158639	0.000000	0.158748		5.542882	0		200		3072000		18	
0.164537	0.000000	0.164646		5.512932	0		200		3072000		18	
0.158638	0.000000	0.158746		4.785556	0		200		3072000		18	
0.164454	0.000000	0.164563		4.790322	0		200		3072000		18	
0.164459	0.000000	0.164562		5.513795	0		200		3072000		18	
0.158658	0.000000	0.158781		4.622990	0		200		3072000		18	
0.164457	0.000000	0.164563		4.790044	0		200		3072000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 4000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 9 ',)
head -c 4000k /dev/urandom > /tmp/tmpn9fp8v2i
Data Size: 4000k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpn9fp8v2i  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 120 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319284	0.000000	0.319392		7.054624	0		200		4096000		18	
0.164468	0.000000	0.164574		6.901487	0		200		4096000		18	
0.158926	0.000000	0.159033		5.851498	0		200		4096000		18	
0.158879	0.000000	0.158983		5.803762	0		200		4096000		18	
0.162986	0.000000	0.163087		5.809492	0		200		4096000		18	
0.164508	0.000000	0.164613		5.810016	0		200		4096000		18	
0.164473	0.000000	0.164574		6.899569	0		200		4096000		18	
0.164463	0.000000	0.164567		6.892696	0		200		4096000		18	
0.158700	0.000000	0.158805		5.803223	0		200		4096000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 5000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 9 ',)
head -c 5000k /dev/urandom > /tmp/tmpmob5ez0j
Data Size: 5000k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpmob5ez0j  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 150 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.330857	0.000000	0.330969		8.358964	0		200		5120000		18	
0.158791	0.000000	0.158898		8.177049	0		200		5120000		18	
0.164516	0.000000	0.164625		8.182312	0		200		5120000		18	
0.158648	0.000000	0.158758		6.804833	0		200		5120000		18	
0.158778	0.000000	0.158890		6.868406	0		200		5120000		18	
0.158756	0.000000	0.158866		6.803439	0		200		5120000		18	
0.164423	0.000000	0.164536		8.179802	0		200		5120000		18	
0.158709	0.000000	0.158819		6.866331	0		200		5120000		18	
0.158662	0.000000	0.158767		6.662361	0		200		5120000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 6000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 9 ',)
head -c 6000k /dev/urandom > /tmp/tmpb7am726c
Data Size: 6000k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpb7am726c  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 180 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319300	0.000000	0.319407		8.006274	0		200		6144000		18	
0.158625	0.000000	0.158734		7.811267	0		200		6144000		18	
0.158816	0.000000	0.158922		7.808325	0		200		6144000		18	
0.164529	0.000000	0.164632		10.972373	0		200		6144000		18	
0.164512	0.000000	0.164618		9.389022	0		200		6144000		18	
0.164520	0.000000	0.164627		9.395499	0		200		6144000		18	
0.158755	0.000000	0.158864		7.809032	0		200		6144000		18	
0.158720	0.000000	0.158824		7.808153	0		200		6144000		18	
0.164482	0.000000	0.164588		9.388693	0		200		6144000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 7000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 9 ',)
head -c 7000k /dev/urandom > /tmp/tmpw7r4pngm
Data Size: 7000k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpw7r4pngm  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 210 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319723	0.000000	0.319829		8.931601	0		200		7168000		18	
0.158666	0.000000	0.158777		10.404877	0		200		7168000		18	
0.158670	0.000000	0.158778		8.762556	0		200		7168000		18	
0.164600	0.000000	0.164710		10.409813	0		200		7168000		18	
0.158882	0.000000	0.158990		8.760088	0		200		7168000		18	
0.158893	0.000000	0.159002		8.762919	0		200		7168000		18	
0.158648	0.000000	0.158755		8.771484	0		200		7168000		18	
0.164471	0.000000	0.164583		10.410991	0		200		7168000		18	
0.164536	0.000000	0.164644		10.409659	0		200		7168000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 8000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 9 ',)
head -c 8000k /dev/urandom > /tmp/tmps4t2nvre
Data Size: 8000k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmps4t2nvre  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 240 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319431	0.000000	0.319543		9.824545	0		200		8192000		18	
0.158621	0.000000	0.158728		11.287017	0		200		8192000		18	
0.158667	0.000000	0.158779		9.654330	0		200		8192000		18	
0.158728	0.000000	0.158835		9.654020	0		200		8192000		18	
0.164384	0.000000	0.164492		11.286443	0		200		8192000		18	
0.164447	0.000000	0.164558		11.295963	0		200		8192000		18	
0.158636	0.000000	0.158742		9.653512	0		200		8192000		18	
0.164360	0.000000	0.164484		11.291335	0		200		8192000		18	
0.164361	0.000000	0.164463		11.291749	0		200		8192000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 9000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 9 ',)
head -c 9000k /dev/urandom > /tmp/tmpj1hrwuk3
Data Size: 9000k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpj1hrwuk3  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 270 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319597	0.000000	0.319707		13.269978	0		200		9216000		18	
0.164559	0.000000	0.164667		12.175935	0		200		9216000		18	
0.164445	0.000000	0.164565		12.165031	0		200		9216000		18	
0.164493	0.000000	0.164597		12.158298	0		200		9216000		18	
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 9000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 5 ',)
head -c 9000k /dev/urandom > /tmp/tmpuidwd297
Data Size: 9000k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpuidwd297  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 270 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.325484	0.000000	0.325589		12.340366	0		200		9216000		18	
0.158840	0.000000	0.158949		12.170279	0		200		9216000		18	
0.158711	0.000000	0.158819		10.534290	0		200		9216000		18	
0.164404	0.000000	0.164510		12.170871	0		200		9216000		18	
0.158638	0.000000	0.158745		10.534892	0		200		9216000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 10000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 9 ',)
head -c 10000k /dev/urandom > /tmp/tmpaqosk_9g
Data Size: 10000k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpaqosk_9g  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 300 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.330823	0.000000	0.330932		13.335863	0		200		10240000		18	
0.164462	0.000000	0.164574		13.251556	0		200		10240000		18	
0.164339	0.000000	0.164451		15.822893	0		200		10240000		18	
0.164548	0.000000	0.164654		13.282267	0		200		10240000		18	
0.158657	0.000000	0.158764		11.413703	0		200		10240000		18	
0.158745	0.000000	0.158850		11.414452	0		200		10240000		18	
0.164542	0.000000	0.164651		13.262760	0		200		10240000		18	
0.158742	0.000000	0.158855		11.270709	0		200		10240000		18	
0.158735	0.000000	0.158846		11.413387	0		200		10240000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 11000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 8 ',)
head -c 11000k /dev/urandom > /tmp/tmpl3gyzfmp
Data Size: 11000k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpl3gyzfmp  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 330 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.319373	0.000000	0.319486		14.920358	0		200		11264000		18	
0.158711	0.000000	0.158839		14.367934	0		200		11264000		18	
0.164628	0.000000	0.164737		14.316797	0		200		11264000		18	
0.158705	0.000000	0.158809		12.419745	0		200		11264000		18	
0.158929	0.000000	0.159035		12.354080	0		200		11264000		18	
0.158677	0.000000	0.158798		12.364228	0		200		11264000		18	
0.164553	0.000000	0.164658		14.625771	0		200		11264000		18	
0.158695	0.000000	0.158803		14.818286	0		200		11264000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 12000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 9 ',)
head -c 12000k /dev/urandom > /tmp/tmpanjww71k
Data Size: 12000k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmpanjww71k  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 360 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.324560	0.000000	0.324670		15.567088	0		200		12288000		18	
0.164636	0.000000	0.164754		16.455283	0		200		12288000		18	
0.158888	0.000000	0.158992		16.612970	0		200		12288000		18	
0.158720	0.000000	0.158826		360.001491	28		000		12288000		0	Operation timed out after 360001 milliseconds with 0 bytes received
0.164605	0.000000	0.164713		15.250850	0		200		12288000		18	
0.158692	0.000000	0.158800		15.436327	0		200		12288000		18	
0.158650	0.000000	0.158757		16.474307	0		200		12288000		18	
0.164390	0.000000	0.164496		16.459043	0		200		12288000		18	
0.158718	0.000000	0.158824		16.520131	0		200		12288000		18	
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 13000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 9 ',)
head -c 13000k /dev/urandom > /tmp/tmp8mmz6m0s
Data Size: 13000k
HTTP: --http3
RUST_LOG=error curl-exp --http3 --insecure --quiche-cc cubic --data-binary @/tmp/tmp8mmz6m0s  https://10.0.1.10:443/ -w "%{time_connect}\t%{time_appconnect}\t%{time_starttransfer}\t\t%{time_total}\t%{exitcode}\t\t%{response_code}\t\t%{size_upload}\t\t%{size_download}\t%{errormsg}\n" --max-time 390 -o /dev/null 2>>error.log
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0.330777	0.000000	0.330889		18.376920	0		200		13312000		18	
0.158716	0.000000	0.158827		17.723470	0		200		13312000		18	
0.164617	0.000000	0.164730		18.445639	0		200		13312000		18	
0.158796	0.000000	0.158908		18.312594	0		200		13312000		18	
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 14000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 4 ',)
head -c 14000k /dev/urandom > /tmp/tmpsakturzb
Data Size: 14000k
HTTP: --http3
/usr/bin/time -f"0		0		0			%e	0	200" /home/gina/quiche-sidecar/target/release/quiche-client --no-verify https://10.0.1.10:443 --body /tmp/tmpsakturzb > /dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0		0		0			24.07	0	200
0		0		0			23.47	0	200
0		0		0			26.06	0	200
0		0		0			23.90	0	200
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 15000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 4 ',)
head -c 15000k /dev/urandom > /tmp/tmpfzz5xi2e
Data Size: 15000k
HTTP: --http3
/usr/bin/time -f"0		0		0			%e	0	200" /home/gina/quiche-sidecar/target/release/quiche-client --no-verify https://10.0.1.10:443 --body /tmp/tmpfzz5xi2e > /dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0		0		0			25.18	0	200
0		0		0			25.65	0	200
0		0		0			25.34	0	200
0		0		0			26.28	0	200
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 16000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 4 ',)
head -c 16000k /dev/urandom > /tmp/tmpwvuxkqyq
Data Size: 16000k
HTTP: --http3
/usr/bin/time -f"0		0		0			%e	0	200" /home/gina/quiche-sidecar/target/release/quiche-client --no-verify https://10.0.1.10:443 --body /tmp/tmpwvuxkqyq > /dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0		0		0			27.29	0	200
0		0		0			25.88	0	200
0		0		0			25.43	0	200
0		0		0			28.62	0	200
*** Stopping 0 controllers

*** Stopping 2 links
..
*** Stopping 0 switches

*** Stopping 3 hosts
h1 h2 r1 
*** Done
max_queue_size = 140 packets
(10.00Mbit 75ms delay 0.00000% loss) (10.00Mbit 75ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) (100.00Mbit 1ms delay 0.00000% loss) *** Configuring hosts
h1 h2 r1 
[sidecar] Setting congestion control to cubic
[sidecar] tso and gso are OFF
[sidecar] Starting the NGINX/Python webserver on h1...
[sidecar] NOT starting the TCP PEP or sidecar
*** h2 : ('python3 mininet/client.py -n 17000k --http 3 --stdout /dev/null --stderr error.log -cc cubic --loss 0 --log-level error -t 4 ',)
head -c 17000k /dev/urandom > /tmp/tmpbpb7_mr3
Data Size: 17000k
HTTP: --http3
/usr/bin/time -f"0		0		0			%e	0	200" /home/gina/quiche-sidecar/target/release/quiche-client --no-verify https://10.0.1.10:443 --body /tmp/tmpbpb7_mr3 > /dev/null
time_connect	time_appconnect	time_starttransfer	time_total	exitcode	response_code	size_upload	size_download	errormsg
0		0		0			29.12	0	200
0		0		0			30.24	0	200
